content/TrueCommand/Administration/Settings/SAMLAD.md

```diff
@@ -0,0 +1,216 @@
+---
+title: "Configuring TrueCommand SAML Service for Active Directory"
+description: "This article provides detailed instructions on setting up SAML service in TrueCommand and Active Directory."
+weight: 40
+tags:
+ - tcsaml
+---
+
+{{< toc >}}
+
+Security Assertion Markup Language (SAML) is a single sign-on (SSO) standard for logging users into applications that require authentication credentials (like GitHub, G-Mail, etc.). 
+Single Sign-on (SSO) works by transferring a user’s known identity to another location that provides services to the user. SAML accomplishes the transfer by exchanging digitally-signed XML documents. 
+
+A SAML configuration requires an Identity Provider (IdP) and Service Provider (SP). Active Directory is an example of an IdP. 
+
+## Activating TrueCommand SAML Service
+
+This article provides instructions for setting up SAML service in TrueCommand and then in Active Directory. 
+
+## Setting up SMAL in TrueCommand
+
+1. Log in to your TrueCommand system (i.e., server, container, VM). 
+   To access the TrueCommand web interface, enter http://*IP:PORT* where *IP:PORT* is the IP address and port number assigned to your TrueCommand system, into a web browser URL field and then press <kbd>Enter</kbd>. 
+
+2. Go to **Settings > Administration**, then click on **Configure** in the **Configuration** widget. 
+   The **Configuration** screen with editable settings displays.
+
+   {{< hint ok >}}
+   The **SSL Options** on the **Configuration** settings form has **Accept self-signed certificates** selected by default. 
+   Keep this setting to be able to use a self-signed certificate. It also helps prevent issues with HTTP versus HTTPS URLs.
+   {{< /hint >}}
+
+3. Enter the URL from Active Directory.
+   Enter http://*ds.yourcompany.net*/FederationMetadata/2007-06/FederationMetadata.xml in the **SAML Identity Provider URL** field, then click **Save**. 
+
```

Comment:
Click on **Configure** in the **Configuration** widget.

Reply:
Save dumps you out so you have to click Configure again to get back into the Configuration screen.